crates/tauri/src/window/mod.rs

```diff
@@ -406,14 +406,12 @@ tauri::Builder::default()
       window.on_menu_event(handler);
     }
 
-    if let Some(effects) = self.window_effects {
-      crate::vibrancy::set_window_effects(&window, Some(effects))?;
-    }
-
     let app_manager = self.manager.manager_owned();
     let window_label = window.label().to_string();
+    let window_ = window.clone();
     // run on the main thread to fix a deadlock on webview.eval if the tracing feature is enabled
     let _ = window.run_on_main_thread(move || {
+      let _ = crate::vibrancy::set_window_effects(&window_, self.window_effects);
```

Comment:
I think we should still check if `self.window_effects` is some, or we'll be calling `clear_effects` unnecessarily